packages/rest/src/types.ts

```diff
@@ -0,0 +1,4 @@
+export interface ApiError {
```

Comment:
Can this be moved to something related to errors? Or maybe rename the file? `types.ts` is quite vague.